quinn-proto/src/config.rs

```diff
@@ -793,6 +826,10 @@ impl ServerConfig {
 
             preferred_address_v4: None,
             preferred_address_v6: None,
+
+            max_incoming: 1 << 16,
+            max_buffer_bytes_per_incoming,
+            max_buffer_bytes_all_incoming: 100 << 20,
```

Comment:
Document/justify these defaults so that it's easier to revisit them in the future if need be.

Reply:
Documented defaults and their reasons in the method docs.